### ISSUE TITLE
clarification on requirements API

### ISSUE THREAD

**Issue by theboyknowsclass** [OfficeDev/office-js-docs-reference#1523]
is this:

`[API set: SharedRuntime 1.1, Mailbox 1.10](https://learn.microsoft.com/en-us/office/dev/add-ins/reference/javascript-api-for-office)`

mean that *BOTH* SharedRuntime 1.1 and Mailbox 1.10 are needed... or *EITHER*

if *BOTH* does that mean this API is restricted to Outlook?

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 19a1bd5f-9f19-f02c-1e0f-761f417eb315
* Version Independent ID: 215c7d97-0cdd-2921-75a3-9875fa685b42
* Content: [OfficeRuntime.Storage interface - Office Add-ins](https://learn.microsoft.com/en-us/javascript/api/office-runtime/officeruntime.storage?view=common-js)
* Content Source: [docs/docs-ref-autogen/office_release/office-runtime/officeruntime.storage.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/live/docs/docs-ref-autogen/office_release/office-runtime/officeruntime.storage.yml)
* Product: **non-product-specific**
* Technology: **add-ins**
* GitHub Login: @o365devx
* Microsoft Alias: **o365devx**

**Comment by AlexJerabek** [comment]
Hi @theboyknowsclass,

Thanks for raising this issue.

This feature was originally created for Custom Functions for Excel, so it only requires the `SharedRuntime 1.1` requirement set there. Given that [Shared Runtime](https://learn.microsoft.com/en-us/javascript/api/requirement-sets/common/shared-runtime-requirement-sets?view=common-js-preview) is only supported by Excel, Word, and PowerPoint, I think it's safe to assume either `SharedRuntime 1.1` or `Mailbox 1.10` is needed.

Let me assign this to @samantharamon to clarify the documentation and provide more context if needed.

**Comment by samantharamon** [comment]
Hi @theboyknowsclass,

Thanks again for reaching out about this. We've clarified the documentation in DefinitelyTyped/DefinitelyTyped#64872. This issue will automatically close once the change is merged. If you need additional help, please don't hesitate to open a new issue on GitHub.
